xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/meta/ListMetaClass.java

```diff
@@ -91,6 +91,13 @@ private void addPresentationMetaProperties()
         sortClass.setPrettyName("Sort");
         sortClass.setValues("none|id|value");
         safeput(sortClass.getName(), sortClass);
+
+
+        StaticListClass freeTextClass = new StaticListClass(this);
+        freeTextClass.setName("freeText");
+        freeTextClass.setPrettyName("Free text");
+        freeTextClass.setValues("forbidden|discouraged|allowed");
```

Comment:
I guess this means that the default for a new class (or when editing an old class) is `forbidden`. Problem is from what I understand this is different from null behavior which seems to be more `discouraged`.

Reply:
AFAICS The default is empty ("------" is selected in the object) which has the same effect as "discouraged". I'll move "discouraged" as the first element, just in case.

Reply:
> AFAICS The default is empty

OK.